docs/configure-rbac-permissions.md

```diff
@@ -1,70 +1,76 @@
 # Configure RBAC permissions
 
-`Microsoft.ApplicationInsights.Kubernetes` uses the service account to query kubernetes information to enhance telemetries. It is important to have proper permissions configured for kubernetes related information like node, pod and so on to be fetched correctly.
+`Microsoft.ApplicationInsights.Kubernetes` uses the service account to query Kubernetes information to enhance telemetries. It is important to have proper permissions configured for Kubernetes-related information like node, pod, and so on to be fetched correctly.
```

Comment:
```suggestion
`Microsoft.ApplicationInsights.Kubernetes` uses the service account to query Kubernetes information to enhance telemetries. It is important to have proper permissions configured for Kubernetes-related resources like Node, Pod, and so on to be fetched correctly.
```
(in K8s API lingo Node, Pod, Deployment etc are called "resources" and spelled starting with a capital letter)

Reply:
Cool